cstool/cstool.c

```diff
@@ -359,6 +363,11 @@ static void usage(char *prog)
 		printf(")\n");
 	}
 
+	if (cs_support(CS_ARCH_ARC)) {
+		printf("        arc         arc little endian\n");
+		printf("        arcbe       arc big endian\n");
+	}
+
```

Comment:
```suggestion
```

Isn't necessary anymore, because you added it above. Also, there was no big endian support in `ARCDisassembler.c`